src/main/resources/templates/db_scripts/schema.json

```diff
@@ -687,6 +687,25 @@
           "removeAccents": true
         }
       ]
+    },
+    {
+      "tableName": "routing_list",
+      "fromModuleVersion": "mod-orders-storage-13.8.0",
+      "withMetadata": true,
```

Comment:
there are chances that the same routing list can be edited by multiple users simultaniously, in the project we use optimistic locking technic to negotiate with this. We add optimistic locking for new tables and also have feature in backlog to add it to an existing tables.
So need to do 2 minor changes here: add "withOptimisticLocking": "failOnConflict" here and also add _version property to routing list schema.
You can use this PR as example: https://github.com/folio-org/mod-organizations-storage/pull/102/files

Reply:
Added both